lessons/Early-Math/translating/langs/en-us/index.adoc

```diff
@@ -0,0 +1,183 @@
+= Translating Between Words and Math
+
+@description{Students learn to model arithmetic expressions in words with a visual tool known as "Circles of Evaluation".}
+
+@lesson-prereqs{}
+
+@keywords{Circle of Evaluation, mathematical expression, equation}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Model an arithmetic expression using @vocab{Circles of Evaluation}.
+* Translate between expressions in words, Circles of Evaluation, and mathematical expressions.
+* Recognize when expressions in words are ambiguous.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's analyze the structure of expressions in words and numbers using Circles of Evaluation.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* This activity involves a card sort. The easiest way to prepare for the card sort is to give each pair of students an envelope containing the three sets of cards. Keep each set (Expressions, Circles of Evaluation, and Words) together with rubber bands or paperclips.
+
+| Key Points For The Facilitator
+|
+* This lesson may challenge students' ideas of math as a subject that is entirely black and white. This theme - that there are oftentimes a variety of completely valid ways of seeing a seemingly simple problem - will emerge again.
+|===
+
+== Circles of Evaluation @duration{30 minutes}
+
+=== Overview
+Students match Circles of Evaluation to mathematical expressions, and then they consider how mathematical expressions in words map onto Circles of Evaluation.
+
+=== Launch
+
+Give each pair of students an envelope and explain that it contains @handout{pages/three-sets-cards.adoc, "three sets of cards"}.
+
+[.lesson-instruction]
+- Look through the first set of cards. What do you notice?
+** _Students should notice that the first set of cards includes @vocab{mathematical expressions}. The expressions on these cards each include the number 15, the number 3, and an operator. Note that because these are expressions - not @vocab{equations} - they do not include an equal sign._
+- Now, look through the second set of cards. What do you notice?
+** _Students should observe that each card includes a circle. They may also notice the position of the numbers and operator within the circle._
+
+This second set of cards includes @vocab{_Circles of Evaluation_}. Circles of Evaluation help students visualize the structure of the mathematical expressions they encounter. For the time being, here’s what students need to know about them:
+
+- Every Circle must have one - and only one! - function, written at the top.
```

Comment:
Are you sure you want to talk about functions here? You could use operation instead, which is a word more commonly used in math books/conversations for this grade level. While they are functions in WeScheme, In pyret all of the things you are referring to as functions are categorized as operators...

Reply:
Hm. This is interesting. I'm not sure I want to talk about functions - but I wonder if we want to preserve that language for when students start using abs-val and negate? 

I'm currently leaning toward putting: "Every Circle must have one - and only one! - function (in this case, an _operator_), written at the top."

What do you think? Maybe @schanzer wants to weigh in?

... I should probably use @vocab to add "function" and "operator" to the Early Math glossary.